### PR TITLE
fix(sandpack-react): build ReactDevTools component

### PR DESCRIPTION
**Pull request by jeetiss**
Upgrades `rollup.config` to build `ReactDevTools` properly and moves types generations to rollup

fix: #712

### CODE REVIEW

File: sandpack-react/package.json [hunk 1]
```diff
@@ -20,8 +20,7 @@
     "prebuild": "yarn run clean",
     "test": "TEST_ENV=true jest . --transformIgnorePatterns \"node_modules/(?!@codemirror)/\" --modulePathIgnorePatterns \"e2e\"",
     "lint": "eslint '**/*.ts?(x)' --fix",
-    "build": "rollup -c && yarn build:types",
-    "build:types": "tsc -p tsconfig.json",
+    "build": "rollup -c --bundleConfigAsCjs",
```
Comment (danilowoz) [inline sandpack-react/package.json:23]: We're making this very same change on https://github.com/codesandbox/sandpack/pull/798

File: sandpack-react/src/components/index.ts [hunk 2]
```diff
@@ -7,3 +7,4 @@ export * from "./Preview";
 export * from "./TranspiledCode";
 export * from "./Tests";
 export * from "./Console";
+export * from "./ReactDevTools";
```
Comment (danilowoz) [inline sandpack-react/src/components/index.ts:10]: `ReactDevTools` is not currently working, that's why it hasn't been exported

**Review by danilowoz — COMMENTED** [review]
Thank you for your contribution, but I don't think we can merge it (see comments)

File: sandpack-react/package.json (hunk 1 above)
Comment (danilowoz) [reply sandpack-react/package.json:23]: Feel free to contribute there too

File: sandpack-react/src/components/index.ts (hunk 2 above)
Comment (jeetiss) [reply sandpack-react/src/components/index.ts:10]: yes, i miss that)

**Pull request closed without merging**